src/connect/component.jsx

```diff
@@ -10,22 +10,55 @@ import {
   loadFraudnet,
   getCSPNonce,
   getDebug,
+  getSessionID,
 } from "@paypal/sdk-client/src";
 import { FPTI_KEY } from "@paypal/sdk-constants";
 
 import { sendCountMetric } from "./sendCountMetric";
 
+const MIN_MAJOR_VERSION = 3;
+const MIN_MINOR_VERSION = 97;
+export const MIN_BT_VERSION = `${MIN_MAJOR_VERSION}.${MIN_MINOR_VERSION}.3-connect-alpha.6.1`; // Minimum for supporting AXO
+
+export function getSdkVersion(version: string | null): string {
+  if (!version) {
+    return MIN_BT_VERSION;
+  }
+  const versionSplit = version.split(".");
+  const majorVersion = Number(versionSplit[0]);
+  const minorVersion = Number(versionSplit[1]);
+
+  if (
+    majorVersion < MIN_MAJOR_VERSION ||
+    (majorVersion === MIN_MAJOR_VERSION && minorVersion < MIN_MINOR_VERSION)
+  ) {
+    sendCountMetric({
+      name: "pp.app.paypal_sdk.connect.init.error.count",
+      event: "error",
+      dimensions: {
+        errorName: "braintree_version_not_supported_error",
+      },
+    });
+
+    throw new Error(
+      `The braintree version used does not support Connect. Please use version ${MIN_BT_VERSION} or above`
+    );
+  }
+
+  return version;
+}
+
 // $FlowFixMe
 export const getConnectComponent = async (merchantProps = {}) => {
-  const cmid = getClientMetadataID();
+  const cmid = getClientMetadataID() || getSessionID();
```

Comment:
If I refresh the page does `getSessionID()` return a unique id each time?  What determines a "session"?

Reply:
Ideally we don't want a unique id.  So if a Buyer goes between checkout page and other pages it would return the same ID upon returning to the checkout page.  Is that how it would work?

Reply:
Gotcha that should be perfect ... sharing w/ our analytics team for visibility as they use this id to identity unique sessions.   Will tag them in our Slack thread [here](https://paypal.slack.com/archives/C04FBV3163E/p1704735353301459?thread_ts=1704383556.640879&cid=C04FBV3163E)